src/bun.js/bindings/KeyObject.cpp

```diff
@@ -381,6 +381,7 @@ JSC::EncodedJSValue KeyObject__createPrivateKey(JSC::JSGlobalObject* globalObjec
                         auto value = pass.value();
                         passphrase.passphrase = const_cast<char*>(value.data());
                         passphrase.passphrase_len = value.length();
+                        pass_holder = std::move(value);
```

Comment:
This is a use-after-move. 

Maybe PrivateKeyPassphrase should have a wrapper class which owns the WTF::CString?

Reply:
fixed here: https://github.com/oven-sh/bun/pull/8797/commits/5be7384f3a4651e11858c95893c4529591e3e0c4